docs/source/advanced/tpu.rst

```diff
@@ -64,8 +64,7 @@ To get a TPU on colab, follow these steps:
 
    .. code-block::
 
-        !curl https://raw.githubusercontent.com/pytorch/xla/master/contrib/scripts/env-setup.py -o pytorch-xla-env-setup.py
-        !python pytorch-xla-env-setup.py --version 1.7 --apt-packages libomp5 libopenblas-dev
+        !pip install cloud-tpu-client==0.10 https://storage.googleapis.com/tpu-pytorch/wheels/torch_xla-1.8-cp37-cp37m-linux_x86_64.whl
```

Comment:
well here the XLA version shall be aligned with PT version, right?

Reply:
The instructions are for Colab & they has updated the runtime to use torch>=1.8.0

Reply:
does this same command work for kaggle?

Reply:
@lezwon Just checked. It's 1.7 for Kaggle notebooks.
We could add `{torch.__version__}` to the URL as a placeholder.